acquia/acquia_cms
# ACMS-000: Removed composer.lock file.

**Motivation**
Remove composer.lock and keep develop branch up to date with latest released packages.

**Proposed changes**
Removed composer.lock file from develop branch

**Alternatives considered**
NA

**Testing steps**
- execute composer install and verify that all packages downloaded with latest release.

**Merge requirements**
